db_client/models/dfce/taxonomy_entry.py

```diff
@@ -19,12 +19,20 @@
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 """
 
+from enum import Enum
 from typing import Mapping, Sequence
 
 from pydantic.config import ConfigDict
 from pydantic.dataclasses import dataclass as pydantic_dataclass
 
 
+class EntitySpecificTaxonomyKeys(str, Enum):
+    """The entity specific taxonomy keys."""
+
+    DOCUMENT = "_document"
+    EVENT = "event_type"
```

Comment:
Hmmm ... okay for now - but arguably this belongs to "_family" ... we just need to make sure that when we finished all the re-jigging it makes sense :)